### ISSUE TITLE
support `git commit`  options within `gitmoji -c`

### ISSUE THREAD

**Issue by AndreasAugustin** [carloscuesta/gitmoji-cli#1040]
### Description of the problem

Currently it is not possible to use `git commit` flags and options when calling `gitmoji -c`.

Because of that some super useful options like `git commit -S`  to sign the commits is not possible to be used within that command.

### Solution

add the needed flags as optional parameters to the already supported flags.

### Alternatives

Currently  it is possible to add some of the flags to the git global settings. E.g. with the example for `-S`

```bash
git config --global commit.gpgsign true
```

This is possible but somehow not nice

### Additional context

For me in special the `-S` option is needed and useful. In many projects I  want and need to sign commits and should be considered as best practice. For more information please consider https://dev.to/andreasaugustin/git-how-and-why-to-sign-commits-35dn

There are many more useful options which are currently suppressed.  Happy to discuss.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/carloscuesta/gitmoji-cli/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/carloscuesta/gitmoji-cli/blob/master/.github/CONTRIBUTING.md).
- [X] Check that there isn't already an issue requesting the same feature.

**Comment by carloscuesta** [comment]
Hey!

Thanks for opening an issue, completely agree, we should support all the same options that `git` accepts as a parameter without having to parse them on the cli. Any unknown option should be forwarded to the `git` binary.

Perhaps we can fix that by using `allowUnknownFlags` and passing those to `git`

https://github.com/sindresorhus/meow

**Comment by AndreasAugustin** [comment]
> Hey!
> 
> Thanks for opening an issue, completely agree, we should support all the same options that `git` accepts as a parameter without having to parse them on the cli. Any unknown option should be forwarded to the `git` binary.
> 
> Perhaps we can fix that by using `allowUnknownFlags` and passing those to `git`
> 
> https://github.com/sindresorhus/meow
> 
> Do you want to raise a pull-request?

Yes, I can raise a PR. Just cannot promise an ETA. Think I will be at least quite busy the next days (but it shouldn't be a huge change)
